apps/photos/src/services/userService.ts

```diff
@@ -493,3 +494,15 @@ export const updateMapEnabledStatus = async (newStatus: boolean) => {
         throw e;
     }
 };
+
+export async function getDisableCFUploadProxyFlag() {
+    try {
+        const featureFlags = (
+            await fetch('https://static.ente.io/feature_flags.json')
+        ).json() as GetFeatureFlagResponse;
+        return featureFlags.disableCFUploadProxy;
+    } catch (e) {
+        logError(e, 'failed to get feature flags');
+        return undefined;
```

Comment:
Why not return a default value here with CF worked as disabled or enabled

Reply:
ahh yes will update